packages/dashboard/src/styles.css

```
@@ -0,0 +1,21 @@
+ * pnpm workspace symlink, a published npm tarball, or hoisted under `.pnpm/`.
+ *
+ * Host apps import this once:
+ *
+ *   @import "@spree/dashboard/styles.css";
+ *
+ * …and add one `@source "<plugin-package-name>";` line per installed plugin.
+ */
+
+@import "@spree/dashboard-ui/styles.css";
+
+@source "@spree/dashboard";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)